electron/app/copyCaptain/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as sqlite3 from 'sqlite3';
+import * as fs from 'fs';
+import copy from 'recursive-copy';
+import log from 'electron-log';
+import { Directories } from './definitions';
+
+// TODO: possibly rethink if this needs to be a class
+// Original idea is that this would take a db path as a instance variable
+// Would also take things like if { overwrite: true } should be used
```

Comment:
@ayc19 let's pair at 7:30 PM PST on this? I think we should refactor this into `SetupCaptain`

and have `chatBro` and `contactsComrade` inherit from it. I'm not sure what instance variables `SetupCaptain` would have.